### PR TITLE
edit profile screen

### PR DESCRIPTION
**Pull request by miaoxudong111**

### CODE REVIEW

File: public/JS/editProfile.js [hunk 1]
```diff
@@ -15,5 +15,22 @@ $(function () {
   })
 
 /*---------------- Socket.on events ----------------*/
+
+$('#changeDisplayNameButton').click(function(){
```
Comment (farfar336) [inline public/JS/editProfile.js]: I suggest moving this part of the code to "Click events" section, which is just above 

"/*---------------- Socket.on events ----------------*/"

**Comment by farfar336** [comment]
Looks good! I have a suggestion:
image
I suggest changing the alert message to "The display name already exists. Choose a different one"

**Comment by farfar336** [comment]
Thanks for making the change! I approve of merging this

**Review by farfar336 — APPROVED** [review]

**Review by SaraK207 — APPROVED** [review]
Looks good. I approve of merging this

**Pull request merged**